lib/engine/ability/README.md

```diff
@@ -154,8 +154,8 @@ This company may not be bought in.
 
 ## purchase_train
 
-Immediately purchases (for normal amount) the currently available depot train
-for the owning corporation.
+Immediately purchases the currently available depot train for the owning corporation.
+- `free`: If true, the train cost is free, otherwase at cost. Default false.
 
```

Comment:
I used purchase_train ability for rocket of china, as it's the closest ability i found. 
I added the free param, but haven't figure out how to use it in view to change the wording there (It still says buy X train for Y cost). 
If it makes more sense i can add a new ability (exchange_train or similiar) instead of using this one

Reply:
This is fine if you update the view accordingly.